app/default_overrides.go

```diff
@@ -261,15 +224,15 @@ func DefaultConsensusConfig() *tmcfg.Config {
 	cfg.RPC.TimeoutBroadcastTxCommit = 50 * time.Second
 	cfg.RPC.MaxBodyBytes = int64(8388608) // 8 MiB
 
-	cfg.Mempool.TTLNumBlocks = 12
-	cfg.Mempool.TTLDuration = 75 * time.Second
+	// TODO: check if priority mempool needed
+	// cfg.Mempool.TTLNumBlocks = 12
+	// cfg.Mempool.TTLDuration = 75 * time.Second
 	cfg.Mempool.MaxTxBytes = 7_897_088
 	cfg.Mempool.MaxTxsBytes = 39_485_440
-	cfg.Mempool.Version = "v1" // prioritized mempool
+	cfg.Mempool.Type = "flood" // flood mempool
```

Comment:
[todo]

figure out the mempool situation, for us the default needs to be v2 (CAT). ideally we have a backup prioritized mempool that's well tested (like our modified fork of the v1 mempool)